doc/release-notes-1.1.0.rst

```diff
@@ -0,0 +1,621 @@
+  * Changed the default security tag to not be the same as nRF Cloud's security tag, to ensure that users do not overwrite their nRF Cloud certificates.
+  * Created a separate nRF Cloud configuration option for the sample.
+  * Added device shadow update to the sample.
+
+Updated libraries
+-----------------
+
+* :ref:`modem_info_readme`:
+
+  * Reworked the architecture to support a parameter storage module.
+  * Increased the available data.
+  * Extended the modem information with IMEI, IMSI, date, and time information.
+  * Changed the modem information to be handled as JSON object instead of strings.
+  * Fixed known bugs.
+
+* :ref:`lib_fota_download`:
+
+  * Add support for performing MCUboot upgrades.
+    To support this, ``CONFIG_SECURE_BOOT=y`` must be set.
+    To send an upgrade, provide the path to both the S0 and the S1 candidate (separated by a space) to the file parameter of :cpp:func:`fota_download_start`.
+    Both candidates are generated by the build system if ``CONFIG_MCUBOOT_BUILD_S1_VARIANT=y`` is set.
+
+* :ref:`lib_aws_fota` and :ref:`lib_aws_jobs`:
+
+  * Implemented fetching of AWS jobs.
+    This allows a device to be updated if it is offline at the time the update is created.
+  * Refactored the code (improved function names, extracted common functionality, re-used topic buffers given to AWS jobs).
+  * Added unit tests for AWS jobs.
+  * Removed device shadow update from the library.
+
+* MQTT library:
+
+  * Dropped the nRF Connect SDK copy of the MQTT library and adopted Zephyr's :ref:`zephyr:mqtt_socket_interface` library instead.
+
+* :ref:`lib_spm`:
+
+  * Added a new non-secure-callable function :cpp:func:`spm_firmware_info`.
+
+* :ref:`lib_nrf_cloud`:
+
+  * Adopted to the new device shadow format.
+
+* at_host (``lib/at_host``):
+
+  * Updated to use a dedicated workqueue instead of the system workqueue.
+  * Miscellaneous fixes and improvements.
+
+* :ref:`at_cmd_parser_readme`:
+
+  * Refactored the library.
+  * Added :ref:`at_notif_readme` dispatcher.
+
+* LTE link control (``include/lte_lc.h``):
+
+  * Added PLMN lock option (default: false).
+  * Added PDN connection authentication option (default: false).
+  * Added modem modes without GPS (LTE-M or NB-IoT only).
+  * Added fallback to secondary LTE network mode (LTE-M/NB-IoT) if the device fails to connect using the primary network mode.
+  * Added a function to get the periodic TAU and active time settings from the current network registration status.
+  * Added a function to get the current functional mode.
+  * Added a function to get the current network registration status (not registered, roaming, registered home network).
+  * Added a function to get and set the system mode (LTE-M, NB-IoT, and GPS).
+  * Other minor improvements and fixes.
+
+
+Updated drivers
+---------------
+
+* :ref:`at_cmd_readme`: (? is that a driver or a library?)
+
+  * Added an option to initialize the driver manually.
```

Comment:
Hasn't that option been in the driver for some time?

Reply:
It was added while v1.1.0 release was ongoing, it didn't make it to the release branch though:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/870

Reply:
You mean to say `1.0.0` @rlubos ? :)

Reply:
@lemrey Correct, thanks for spotting